Pipfile.lock

```diff
@@ -266,7 +287,6 @@
                 "sha256:3c4b1d770fa509aaad997ca147bd3533b730c3f6c48290b69a4265072c465522",
                 "sha256:4ee4f24cbea5e18e1fefdfccb043e819caf483d16d08e39cb6df5d18b0407275"
             ],
-            "index": "pypi",
```

Comment:
I haven't wrapped my head around Pipfile's yet, but this jumped out to me. Could you explain it?

Reply:
I can't, I'm mostly ignoring the `Pipfile.lock` so far. :-/

Reply:
okay :(